### ISSUE TITLE
Some items not visible in JEI and creative tabs and not craftable

### ISSUE THREAD

**Issue by ghost** [rwtema/extrautilities_old_issues#1038]
mc 1.10.2
forge build 2099
ExtraUtilities2 1.1.0c
Other mods installed: TConstruct, EnderIO & Refined Storage

See attached screenshot. Some items (like generators, conveyor belt etc.) are neither visible in JEI nor creative tabs. Trying to craft them fails.
![2016-10-14_13 25 22](https://cloud.githubusercontent.com/assets/22838247/19388246/db6c790a-921e-11e6-9081-de71d3a778cd.png)

**Comment by Partonetrain** [comment]
That's because they don't exist.

**Comment by DatrixTHLK** [comment]
Sadly, that is true. Why they have made the items BEFORE making the recipes for them is beyond me.

**Comment by Partonetrain** [comment]
Both of you realize RWTema doesn't check this, right?

**Comment by DatrixTHLK** [comment]
Yes, since 2 years ago. Such potential for this mod. Oh well, I'm just going to unsunscribe from this github issues page
